problems/two_sum.rb

```diff
@@ -1,14 +1,15 @@
 class Array
-  # Write a method, `Array#two_sum`, that finds all pairs of positions where the
-  # elements at those positions sum to zero. The method should return a nested 
+  # Define a method `two_sum`, that finds all 
```

Comment:
Should be `Array#two_sum` or otherwise specify that it's monkey patched onto the Array class